.github/workflows/test.yml

```diff
@@ -0,0 +1,76 @@
+name: Test with Dockerfiles
+on:
+  push:

```

Comment:
Why not just on PRs? As we have protected master branch, we don't need to use minutes on testing code after merging it.
Workflow for pushing would also trigger prod deploy, so it'll better to keep them separate.